package.json

```diff
@@ -40,76 +40,75 @@
+    "@angular/platform-browser": "~8.2.14",
+    "@angular/platform-browser-dynamic": "~8.2.14",
+    "@angular/router": "~8.2.14",
+    "@ng-select/ng-select": "^3.7.3",
     "c3": "~0.7.1",
-    "core-js": "~2.4.1",
     "evaporate": "^2.1.4",
     "js-sha256": "^0.9.0",
     "moment-timezone": "^0.5.23",
     "ng-inline-svg": "^8.3.0",
     "ng2-dragula": "^1.5.0",
     "ngx-tooltip": "0.0.9",
     "pikaday": "^1.8.0",
-    "rxjs": "~6.0.0",
+    "rxjs": "~6.5.5",
     "scss-bundle": "~2.4.0",
     "spark-md5": "^3.0.0",
     "stickyfilljs": "^2.1.0",
-    "zone.js": "~0.8.4"
+    "zone.js": "~0.9.1"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.13.0",
-    "@angular-devkit/build-ng-packagr": "~0.8.0",
-    "@angular-devkit/core": "~7.3.5",
-    "@angular/cli": "~7.3.3",
-    "@angular/compiler-cli": "~7.2.0",
-    "@angular/language-service": "~7.2.0",
+    "@angular-devkit/build-angular": "~0.803.26",
+    "@angular-devkit/build-ng-packagr": "^0.901.7",
+    "@angular-devkit/core": "^8.3.26",
+    "@angular/cli": "~8.3.26",
+    "@angular/compiler-cli": "~8.2.14",
+    "@angular/language-service": "~8.2.14",
     "@babel/core": "~7.3.4",
-    "@storybook/addon-a11y": "^5.0.11",
-    "@storybook/addon-actions": "~5.0.11",
-    "@storybook/addon-backgrounds": "~5.0.11",
-    "@storybook/addon-centered": "~5.0.11",
-    "@storybook/addon-knobs": "~5.0.11",
-    "@storybook/addon-links": "~5.0.11",
-    "@storybook/addon-notes": "~5.0.11",
-    "@storybook/addon-options": "~5.0.11",
-    "@storybook/addon-storysource": "~5.0.11",
-    "@storybook/addon-viewport": "^5.0.11",
-    "@storybook/addons": "~5.0.11",
-    "@storybook/angular": "~5.0.11",
+    "@storybook/addon-a11y": "^5.3.19",
+    "@storybook/addon-actions": "~5.3.19",
+    "@storybook/addon-backgrounds": "~5.3.19",
+    "@storybook/addon-centered": "~5.3.19",
+    "@storybook/addon-knobs": "~5.3.19",
+    "@storybook/addon-links": "~5.3.19",
+    "@storybook/addon-notes": "~5.3.19",
+    "@storybook/addon-options": "~5.3.19",
+    "@storybook/addon-storysource": "~5.3.19",
+    "@storybook/addon-viewport": "^5.3.19",
+    "@storybook/addons": "~5.3.19",
+    "@storybook/angular": "~5.3.19",
     "@types/c3": "~0.4.41",
     "@types/jest": "^24.0.11",
     "@types/jsdom": "^12.2.3",
     "@types/moment-timezone": "^0.5.12",
     "@types/node": "~8.9.4",
     "@types/pikaday": "^1.7.4",
-    "@types/storybook__addon-actions": "~3.4.2",
-    "@types/storybook__addon-knobs": "~5.0.0",
+    "autoprefixer": "^8.6.5",
     "babel-loader": "~8.0.5",
-    "codelyzer": "~4.5.0",
+    "codelyzer": "^5.2.2",
     "css-loader": "~2.1.1",
     "dotenv": "~2.0.0",
     "i": "~0.3.6",
-    "jest": "^24.7.1",
-    "jest-preset-angular": "^7.0.1",
-    "ng-packagr": "~4.7.1",
+    "jest": "^25.5.4",
+    "jest-preset-angular": "^8.2.0",
+    "ng-packagr": "^9.1.5",
     "npm": "~6.9.0",
     "protractor": "~5.4.0",
+    "sass-loader": "^7.3.1",
```

Comment:
storybook wants to use sass-loader@8, but it's not compatible, throwing a webpack config error